teamcity: failed tests on master: check/TestStyle, check/TestStyle/TestErrCheck, check/TestStyle/TestReturnCheck, check/TestStyle/TestMetacheck, acceptance/TestDockerCLI, acceptance/TestDockerCLI/test_server_restart.tcl

The following tests appear to have failed:

[#235657](https://teamcity.cockroachdb.com/viewLog.html?buildId=235657):

```
--- FAIL: check/TestStyle (26.180s)
null
--- FAIL: check/TestStyle/TestErrCheck (7.940s)
style_test.go:574: err=exit status 2, stderr=/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica_test.go:4458:21: too few arguments in call to stopper.Stop
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica_test.go:4468:25: cannot use &ts (value of type *github.com/cockroachdb/cockroach/pkg/util/hlc.Timestamp) as github.com/cockroachdb/cockroach/pkg/util/hlc.Timestamp value in assignment
	error: failed to check packages: could not type check: couldn't load packages due to errors: github.com/cockroachdb/cockroach/pkg/storage
--- FAIL: check/TestStyle/TestReturnCheck (10.690s)
style_test.go:598: err=exit status 1, stderr=/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica_test.go:4458:21: too few arguments in call to stopper.Stop
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica_test.go:4468:25: cannot use &ts (value of type *github.com/cockroachdb/cockroach/pkg/util/hlc.Timestamp) as github.com/cockroachdb/cockroach/pkg/util/hlc.Timestamp value in assignment
--- FAIL: check/TestStyle/TestMetacheck (7.560s)
style_test.go:777: err=exit status 1, stderr=/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica_test.go:4458:21: too few arguments in call to stopper.Stop
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica_test.go:4468:25: cannot use &ts (value of type *github.com/cockroachdb/cockroach/pkg/util/hlc.Timestamp) as github.com/cockroachdb/cockroach/pkg/util/hlc.Timestamp value in assignment
	couldn't load packages due to errors: github.com/cockroachdb/cockroach/pkg/storage
--- FAIL: acceptance/TestDockerCLI (158.000s)
test_log_scope.go:79: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI500337515
test_log_scope.go:62: use -show-logs to present logs inline
--- FAIL: acceptance/TestDockerCLI/test_server_restart.tcl (36.920s)
cli_test.go:73: non-zero exit code: 1

```
Please assign, take a look and update the issue accordingly.

--- a/pkg/storage/replica_test.go
+++ b/pkg/storage/replica_test.go
@@ -4455,7 +4455,7 @@ func TestPushTxnQueryPusheeHasNewerVersion(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	tc := testContext{}
 	stopper := stop.NewStopper()
-	defer stopper.Stop()
+	defer stopper.Stop(context.TODO())
 	tc.Start(t, stopper)
 
 	key := roachpb.Key("key")
@@ -4465,7 +4465,7 @@ func TestPushTxnQueryPusheeHasNewerVersion(t *testing.T) {
 	pushee.Sequence = 2
 	ts := tc.Clock().Now()
 	pushee.Timestamp = ts
-	pushee.LastHeartbeat = &ts
+	pushee.LastHeartbeat = ts
 
 	pusher := newTransaction("test", key, 1, enginepb.SERIALIZABLE, tc.Clock())
 	pusher.Priority = 2

